components/gitpod-cli/cmd/env.go

```diff
@@ -213,6 +227,11 @@ func setEnvs(ctx context.Context, scopeUser bool, args []string) error {
 		g.Go(func() error {
 			err = result.client.SetEnvVar(ctx, v)
 			if err != nil {
+				if ferr, ok := err.(*jsonrpc2.Error); ok && ferr.Code == http.StatusForbidden && setEnvScope == envScopeUser {
```

Comment:
When could this happen?

Reply:
Whenever you try to update a user-scoped env var, but it does not exist, yet.
See the discussions on the referenced issue for more details.